joomla/joomla-cms
# [4.0] [mod_tags_similar] Always render module layout

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/20810. Redo of https://github.com/joomla/joomla-cms/pull/20811 for 4.0.

### Summary of Changes

Moves early return when no items are found to the layout. This way the layout is always rendered, allowing users to show custom content when no items are found.
Also removes previously unused message.

### Testing Instructions

On a site without tagged items publish `Tags - Similar` module.
Create module's layout override containing some custom code/message.
Check if the module is displayed.

### Expected result

Module is displayed.

### Actual result

Module not displayed.

### Documentation Changes Required
Probably not.
